content/spin/install.md

```diff
@@ -108,6 +97,14 @@ $ cd spin && make build
 $ ./target/release/spin --help
 </code></pre>
 
+> Please note: On a fresh Linux installation, you will also need the standard build toolchain (`gcc`, `make`, etc.), the SSL library headers, and on some distributions you may need `pkg-config`. For example, on Debian-like distributions, including Ubuntu, you can install the standard build toolchain with a command like this:
```

Comment:
This would also be needed for the Using Cargo to Install Spin method.

Reply:
I might have to re-word a little to cater for Windows users (as not to make it too Linux focussed when referring to cargo)

Reply:
Ooh good catch

Reply:
As discussed, I agree, catering for the alternate platforms can go over to the contributing to Spin page, which is capable of diving into a bit more detail and providing good context. about installing from source.